### ISSUE TITLE
Lib Keyboard U+00B2

### ISSUE THREAD

**Issue by iandoug** [alerque/libertinus#451]
Hi

According to the Wikipedia picture, the French keyboard has a  superscript 2 U+00B2 where US ANSI has ~`.
https://en.wikipedia.org/wiki/AZERTY#/media/File:KB_France.svg

This is missing from Lib Keyboard.

Cheers, Ian

**Comment by ChiefMikeK** [comment]
If this is actually superscript then better to place symbol in the upper right quadrant?
![sup2_keycap](https://user-images.githubusercontent.com/20070053/116780370-2d983000-aa31-11eb-82a4-63dd4beb005a.png)

**Comment by alerque** [comment]
No. The upper right quadrant of the keycap design indicates a 4th level shift (bottom left is base key, top left is shift, bottom right is iso-3rd level shift, top right is 4th (usually shift+iso3). How these are accessed varies, but that's why there are different positions on keycaps, not because of super/subscript status.

This glyph if added should be in the lower-left quadrant, but it can be a small superscript in the top-right of the lower-left quadrant. Note this is what is shown in the layout linked above.
